packages/element/src/react.js

```diff
@@ -106,6 +107,11 @@ export { Fragment };
  */
 export { isValidElement };
 
+/**
+ * @see https://reactjs.org/docs/react-api.html#reactmemo
```

Comment:
It should be included in CHANGELOG that we now expose `memo` as well. 

> React.memo is a higher order component. It’s similar to React.PureComponent but for function components instead of classes.

Are we going to promote it over `pure` HOC we offer in `@wordpress/compose` package? Should we refactor `pure` to use `memo` internally? /cc @youknowriad

Reply:
I think there are differences where pure applies to any component and memo only to function components?